README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_growth_table_DOM/)
+    - [DEMO LINK](https://Uandrik.github.io/js_growth_table_DOM/)
```

Comment:
Add this demo link to the PR description 
<img width="963" alt="image" src="https://github.com/user-attachments/assets/dd544598-ed69-4e19-8026-a9c895f40c8d" />